sql-plugin/src/main/scala/com/nvidia/spark/rapids/TypeChecks.scala

```diff
@@ -643,7 +643,7 @@ object CaseWhenCheck extends ExprChecks {
  * This is specific to WidowSpec, because it does not follow the typical parameter convention.
  */
 object WindowSpecCheck extends ExprChecks {
-  val check: TypeSig = TypeSig.commonCudfTypes
+  val check: TypeSig = TypeSig.commonCudfTypes + TypeSig.DECIMAL
```

Comment:
This says that we should be able to partition by decimal columns and order by them as well, but we have no tests for that.  Could we update/add a few tests to verify that we are covering this use case?

Reply:
I think existing tests has already covered `partition` and `order`.
 In unit test,  `windowAggregationTesterForDecimal` method will cast column for partition and column for order to decimal before window aggregation.
In integration test, `test_multi_types_window_aggs_for_rows_lead_lag` method covers `LEAD`/`LAG` operations, which includes decimal type in both `part_and_order_gens` and `data_gens`.